inference-engine/tests/functional/shared_test_classes/src/single_layer/pooling.cpp

```diff
@@ -107,7 +107,7 @@ std::string MaxPool8LayerTest::getTestCaseName(const testing::TestParamInfo<maxP
     ngraph::op::PadType padType;
     ngraph::op::RoundingType roundingType;
     ngraph::element::Type indexElementType;
-    int32_t axis;
+    int64_t axis;
```

Comment:
you should fix earlier commits with `git commit --fixup` + `git rebase`, changing same piece of code back and forth complicates review and git history. same for test of changes in this commit